integration_tests/src/main/python/array_test.py

```diff
@@ -99,10 +99,15 @@ def test_orderby_array_of_structs(data_gen):
 def test_array_contains(data_gen):
     arr_gen = ArrayGen(data_gen)
     lit = gen_scalar(data_gen, force_no_nulls=True)
-    assert_gpu_and_cpu_are_equal_collect(lambda spark: two_col_df(
-        spark, arr_gen, data_gen).select(array_contains(col('a'), lit.cast(data_gen.data_type)),
-                                         array_contains(col('a'), col('b')),
-                                         array_contains(col('a'), col('a')[5])), no_nans_conf)
+
+    def get_input(spark):
```

Comment:
nit: I always feel much better about a fixes like this if the tests weren't changed. They look to be identical to before, but with a lot of formatting changes.

Reply:
Thanks for reviewing, @revans2.
I'd be willing to undo this change, if we could fix the original formatting. I found the previous formatting very hard to parse.

Reply:
Like I said it is a nit. If you have strong feelings just keep it.